pkgs/by-name/lo/local-ai/package.nix

```diff
@@ -468,7 +475,7 @@ let
     postFixup = ''
       wrapProgram $out/bin/${pname} \
     '' + lib.optionalString with_cublas ''
-      --prefix LD_LIBRARY_PATH : "${cudaPackages.libcublas}/lib:${cudaPackages.cuda_cudart}/lib:/run/opengl-driver/lib" \
+      --prefix LD_LIBRARY_PATH : "${lib.getLib libcublas}/lib:${cuda_cudart}/lib:/run/opengl-driver/lib" \
```

Comment:
- `"${addDriverRunpath.driverLink}/lib"` instead of `/run/...` (in case we change the path in future)
- Since the entire line is modified, you could replace this with `lib.makeLibraryPath`

Reply:
Thanks, I added your proposals in an additional commit.